EmbeddedSocialClient/sdk/src/main/java/com/microsoft/embeddedsocial/ui/activity/PopularActivity.java

```diff
@@ -5,21 +5,15 @@
 
 package com.microsoft.embeddedsocial.ui.activity;
 
-import android.support.v4.app.Fragment;
-import android.support.v4.view.PagerAdapter;
-
+import com.microsoft.embeddedsocial.ui.activity.base.BaseActivity;
 import com.microsoft.embeddedsocial.ui.fragment.FeedViewMenuFragment;
-import com.microsoft.embeddedsocial.base.function.Producer;
-import com.microsoft.embeddedsocial.data.model.TopicFeedType;
 import com.microsoft.embeddedsocial.sdk.R;
-import com.microsoft.embeddedsocial.ui.activity.base.BaseTabsActivity;
 import com.microsoft.embeddedsocial.ui.fragment.PopularFeedFragment;
```

Comment:
should imports be in some kind of order, such as alphabetical?

Reply:
Android Studio should automatically organize imports according to the Android conventions (which I believe are alphabetical, but case sensitive, and grouped by namespace) if you right click on file and select "Reformat code" and check "Optimize imports".  This will also remove unused imports, and group some with `*` if needed.

Reply:
I'll submit a separate PR for fixing imports and white space throughout the SDK.